src/main/java/de/tum/in/www1/artemis/web/rest/errors/BadRequestException.java

```diff
@@ -0,0 +1,19 @@
+package de.tum.in.www1.artemis.web.rest.errors;
+
+import java.io.Serial;
+
+import org.zalando.problem.AbstractThrowableProblem;
+import org.zalando.problem.Status;
+
+/**
+ * Simple exception with a message, that returns a Bad Request code.
+ */
+public class BadRequestException extends AbstractThrowableProblem {
```

Comment:
Why exactly do we need this?
What about ``BadRequestAlertException`` which has a ``skipAlert`` which would also be able to support i18n, or be able to skip the alert if you don't want it...

Reply:
That is part of a discussion we've had earlier. Handling server side errors in the client of Artemis is vastly inconsistent. Right now, we have a lot of services sending errors in response headers that spawn alerts; I also think that this is a questionable approach, but we'd have to globally change it
Anyways, wouldn't you be able to just use the ``BadRequestAlertException`` with the ``skipAlert`` option rather than creating a new general-purpose exception that is only used once? :)

Reply:
Done